src/daemon/rpmostreed-os.c

```diff
@@ -1136,7 +1240,7 @@ os_handle_set_initramfs_state (RPMOSTreeOS *interface,
     {
       const char *client_address;
       client_address = rpmostreed_transaction_get_client_address (transaction);
-      rpmostree_os_complete_pkg_change (interface, invocation, NULL, client_address);
+      rpmostree_os_complete_set_initramfs_state (interface, invocation, client_address);
```

Comment:
Break out fixups like this to prep patches?

Reply:
Yeah, there's definitely a lot I have to split out from this. Will get started on that today.